ui/pages/login/index.js

```diff
@@ -39,9 +39,16 @@ export function saveToVisitedOrgs(domain, baseDomain, orgName) {
       name: orgName,
     })
 
+    // set the cookie domain to a general base domain
+    let cookieDomain = window.location.host
+    let parts = cookieDomain.split('.')
+    if (parts.length > 2) {
+      cookieDomain = parts.slice(-2).join('.') // join the last two parts of the domain
```

Comment:
In this case, would it be better to set it to all but the first part? e.g. `myorg.dev.test.com` may not work here

Reply:
The "last 2 parts" here would be `test.com` which in my tests gets the behavior we want of sending the cookie on any domain at `test.com/login/organizations`.

![Screen Shot 2022-10-04 at 10 40 14 AM](https://user-images.githubusercontent.com/5853428/193835129-cbca374f-e286-43b9-bcd9-ffc3c5e89a35.png)

Reply:
Fixed this to just remove the first part that is the org subdomain, this will work in dev and prod as you pointed out